app/controllers/conditions_controller.rb

```diff
@@ -0,0 +1,27 @@
+class ConditionsController < ApplicationController
+  def condition
```

Comment:
一覧に相当するところなので `index` で良いかなと思いました
```suggestion
  def index
```